### PR TITLE
Rework mk_sdcard_image script

### PR DESCRIPTION
**Pull request by rshym**
Some improvements for script:
- remove flood messages
- optimize usage of losetup
- redefine hard coded variable

### CODE REVIEW

File: doc/mk_sdcard_image.sh [hunk 1]
```diff
@@ -79,19 +79,20 @@ partition_image()
...
-	sudo partprobe $android_disk || true
+	sudo parted $temp_dev -s mklabel gpt || true
+	sudo parted $temp_dev -s mkpart xvda1 ext4 1MB  3148MB || true
+	sudo parted $temp_dev -s mkpart xvda2 ext4 3149MB  3418MB || true
+	sudo parted $temp_dev -s mkpart xvda3 ext4 3419MB  3420MB || true
+	sudo parted $temp_dev -s mkpart xvda4 ext4 3421MB  4421MB || true
+	sudo parted $temp_dev -s print
+	sudo partprobe $temp_dev || true
+
+    sudo losetup -d $temp_dev
```
Comment (andr2000) [inline doc/mk_sdcard_image.sh]: indentation?

File: doc/mk_sdcard_image.sh [hunk 2]
```diff
@@ -3,6 +3,7 @@
 MOUNT_POINT="/tmp/mntpoint"
 CUR_STEP=1
 DOMA_PART_N=p3
+DEFAULT_IMAGE_SIZE_GIB=7
```
Comment (andr2000) [inline doc/mk_sdcard_image.sh]: why 7? not 8 ;)

File: doc/mk_sdcard_image.sh (hunk 1 above)
Comment (rshym) [reply doc/mk_sdcard_image.sh]: Fixed.

File: doc/mk_sdcard_image.sh (hunk 2 above)
Comment (rshym) [reply doc/mk_sdcard_image.sh]: Reworked.
DEFAULT_IMAGE_SIZE_GIB is defined inside partition_image() as sum of sizes of all partitions.

**Comment by rshym** [comment]
Comments are addressed.
"Reviewed-by" is added.

**Comment by rshym** [comment]
@iartemenko could you please merge?

**Pull request merged**